deploy/kubeturbo-operator/config/crd/bases/charts.helm.k8s.io_kubeturboes.yaml

```diff
@@ -165,9 +166,6 @@ spec:
                   stitchuuid:
                     description: Identify if using uuid or ip for stitching
                     type: boolean
-                  cleanupSccImpersonationResources:
```

Comment:
I think you need to rebase with 8.8.2 crd instead of 8.8.1, since this command line arg was newly introduced from 8.8.2 - https://github.com/turbonomic/kubeturbo/pull/808

Reply:
Thanks @SumanthKodali999. I updated the CRD to point to version [8.8.2](https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/kubeturbo-certified/8.8.2/manifests/kubeturbos.charts.helm.k8s.io.crd.yaml)